File: lab-2/src/moves/Swagger.java
package moves;

import ru.ifmo.se.pokemon.*;

public class Swagger extends StatusMove {
    public Swagger() {
        super(Type.NORMAL, 0, 85);
    }


    protected void applyOppEffects(Pokemon p) {
        p.confuse();
        p.setMod(Stat.ATTACK, 2);
    }

    protected java.lang.String describe() {
        return ("is using Swagger");
    }
}

File: lab-2-kotlin/src/moves/Swagger.kt
package moves

import ru.ifmo.se.pokemon.*

class Swagger : StatusMove(Type.NORMAL, 0.0, 85.0) {
    override fun applyOppEffects(p: Pokemon) {
        p.confuse()
        p.setMod(Stat.ATTACK, 2)
    }

    override fun describe(): String {
        return "is using Swagger"
    }
}